editor/scene_tree_dock.cpp

```diff
@@ -2416,6 +2512,15 @@ void SceneTreeDock::_tree_rmb(const Vector2 &p_menu_pos) {
 		menu->add_icon_shortcut(get_theme_icon("Collapse", "EditorIcons"), ED_GET_SHORTCUT("scene_tree/expand_collapse_all"), TOOL_EXPAND_COLLAPSE);
 		menu->add_separator();
 
+		if (profile_allow_editing) {
+			menu->add_shortcut(ED_GET_SHORTCUT("scene_tree/cut_node"), TOOL_CUT);
+			menu->add_shortcut(ED_GET_SHORTCUT("scene_tree/copy_node"), TOOL_COPY);
```

Comment:
Is there a reason why copy and cut menu items are not available for multiple node selection like duplicate? Multiple node copy/cut seems to work fine when using keyboard shortcuts.

Reply:
This is a leftover from the time when it wasn't possible. I implemented the multinode copy and forgot to change this part.